grad_buymenu/definitions/suppliesBlufor.sqf

```diff
@@ -84,23 +85,25 @@ _uh1y = [
       2,
       0
 ];
-_m113 = [
-      ["rhsusf_m113_usarmy"],
+_lav25 = [
+      ["CUP_B_LAV25M240_USMC"],
       [],
       3,
-      1500,
+      2000,
       1,
       [],
       {
       clearWeaponCargoGlobal (_this select 0);
       clearItemCargoGlobal (_this select 0);
       clearBackpackCargoGlobal (_this select 0);
       clearMagazineCargoGlobal (_this select 0);
+      (_this select 0) disableTIEquipment true;
+      (_this select 0) removeMagazinesTurret ["CUP_210Rnd_TE1_Red_Tracer_25mm_M242_HE",[0]];
```

Comment:
ich seh hier iwie zehntausendmal `(_this select 0)` . Wollt ihr euch das nicht in ne extra Variable packen? Machts leserlicher wenn da sowas steht wie:

```
_car = _this select 0;
clearMagazineCargoGlobal _car;
_car disableTIEquipment true;
['AddCargoByClass', _car, ['ACE_track', _this select 0, 2]] call ace_common_fnc_targetEvent;
// etc
```